tests/test_tables.py

```diff
@@ -0,0 +1,206 @@
+from .base import BaseTinyFatTest
+from .utils import match_all_elements
+from tinyfatdb import TinyFatDB, TinyFatTable
+from tinydb import TinyDB
+from tinydb import Query as Q
+
+a_ends_with_2 = Q().a.test(lambda a: a.endswith("2"))
+
+
+########################################################################
+class TableAndModelTestSuite:
```

Comment:
I am not a huge fan of how this is being used. I believe that it will be useful to make sure that the code is concurrent, but I think that these tests are so bare bones that it may not catch instances where this code conflicts with the `tinydb` code.

While (a) these tests are fast, and (b) there are not enough tests in this package's current state to worry about efficiency, it still is offsetting to see each of these tests run twice.

Ultimately, it is your call whether you want to keep this code and run each test twice to test the concurrency. I do not think that it is totally necessary. If we do decide to keep it, I think that we should make sure that we are testing concurrency in all areas (not just the tables), and in depth.